Makefile

```diff
@@ -34,8 +43,21 @@ build: fmt vet
 vet:
 	$(GO) vet ./...
 
+lint:
+	if [ $$($(GO_LINT) ./... | wc -l) != 0 ]; then \
+	        if $(is_ci); then \
+	                echo "FATAL: you should run 'make lint' on your local system"; \
+	                exit 1; \
+	        fi; \
+	fi
+
 fmt:
-	$(GO) fmt ./...
+	if [ $$($(GO) fmt ./... | wc -l) != 0 ]; then \
```

Comment:
Why this change? don't we want fmt to just silently format the files for us (like it does today)?